lib/pages/screening_page.dart

```diff
@@ -15,38 +41,74 @@ class Screening extends StatelessWidget {
         leadingAction: () => Navigator.pop(context),
       ),
       body: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          crossAxisAlignment: CrossAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'Will be a list of things to look for in a potential victim of family violence.',
-              style: new TextStyle(
-                color: purple.shade500,
-                fontSize: 20
-              )
-            ),
-            ListView(
-              shrinkWrap: true,
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
               children: <Widget>[
-                ListTile(
-                  leading: Icon(Icons.label),
-                  title: Text('Sign 1'),
+                Padding(
+                  padding: const EdgeInsets.all(10.0),
+                  child: Text(
+                    'Here is a list of questions to ask someone to determine '
+                        'if the person may have been a victim of domestic'
+                        ' violence.',
+                    style: new TextStyle(
+                      color: purple.shade500,
+                      fontSize: 25,
+                    ),
+                  ),
+                ),
+                Divider(
+                  color: purple.shade500,
+                  height: 5.0,
+                  thickness: 5.0,
                 ),
-                ListTile(
-                  leading: Icon(Icons.label),
-                  title: Text('Sign 2'),
+                FlatButton(
+                  child: Text('Clear'),
+                  onPressed: () => {
+                   Screening.screeningTips.forEach((tip) => tip.selected = false)
```

Comment:
Missing a setState() call here to update the page so the button does not work as intended. Simple fix.

setState(() { Screening.screeningTips.forEach((tip) => tip.selected = false); });

** sidenote though, usually state variables are declared in the state class, not the widget so you should transfer the the screeningTips array to the state class. otherwise it looks good.

Reply:
It is asking me to update the SDK constraints in order to do this, as "Set literals weren't supported until version 2.2, but this code is required to be able to run on earlier versions"

Reply:
Also, is this how it should go in?
` 
FlatButton(
                  child: Text('Clear'),
                  onPressed: () => {
                  setState(() { Screening.screeningTips.forEach((tip) => tip.selected = false); })
                  },
                ),
`

Reply:
FlatButton( child: Text('Clear'), onPressed: () => { setState(()  Screening.screeningTips.forEach((tip) => tip.selected = false); ) }, ),

Reply:
FlatButton( child: Text('Clear'), onPressed: () => setState(() { Screening.screeningTips.forEach((tip) => tip.selected = false); )),